### PR TITLE
Text utilities list components

### PR DESCRIPTION
**Pull request by nisargpawade**
- changed color of text inputs for more visibility
- made text utilities
  - headings
  - texts
  - font weight
  - gray text and centered text
- unordered simple list
- unordered stacked list

image
image
image

### CODE REVIEW

File: components/list/list.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+.list-container li:hover {
+    font-weight: 500;
+}
+
+.list-stacked-container {
+    list-style: none;
+    max-width: 60rem;
+    border: 1px solid ;
+    border-radius: 5px;
```
Comment (theVivek-kumar) [inline components/list/list.css:29]: use  rem

Comment (theVivek-kumar) [inline components/list/list.css:29]: use rem

File: components/text-utilities/text-utilities.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
+h1 {
+    font-size: 3.2rem;
+    margin: 8px 0 5px 0;
```
Comment (theVivek-kumar) [inline components/text-utilities/text-utilities.css:3]: use rem

File: docs.html [hunk 3]
```diff
@@ -54,6 +54,8 @@
             <a class="sidebar-link" href="./components/image/image.html">Images</a>
           </li>
           <li class="sidebar-li"><a class="sidebar-link" href="./components/input/input.html">Input</a></li>
```
Comment (theVivek-kumar) [inline docs.html:56]: class names are good to me

**Review by theVivek-kumar — CHANGES_REQUESTED** [review]

File: components/alert/alert.html [hunk 4]
```diff
@@ -99,6 +101,7 @@ <h2>Simple Alert</h2>
             <i class="fas fa-info-circle"></i><span>This is an info alert</span>
           </div>
         </div>
+        
         <pre><code class="display-code language-html">
           &lt;div class="alert"&gt;
             &lt;span&gt;This is an alert&lt;/span&gt;
```
Comment (manojsarna) [inline components/alert/alert.html:107]: what is this?

File: components/card/card.html [hunk 5]
```diff
@@ -111,7 +113,7 @@ <h2>Simple Card</h2>
                     &lt;div class="card-text margin-l-1"&gt;
                       &lt;p&gt;&lt;b&gt;Product Name&lt;/b&gt;&lt;/p&gt;
                       &lt;small&gt;Company Name&lt;/small&gt;
-                      &lt;p class="p-small"&gt;Product description and other text&lt;/p&gt;
+                      &lt;p class="p-sm"&gt;Product description and other text&lt;/p&gt;
                     &lt;/div&gt;
                     &lt;div class="card-btns"&gt;
                       &lt;button class="btn"&gt;Buy&lt;/button&gt;
```
Comment (manojsarna) [inline components/card/card.html:119]: do check this

File: components/input/input.css [hunk 6]
```diff
@@ -7,22 +7,25 @@
...
+    font-weight: 500;
 }
 
 .input-warning {
     color: var(--warning-dark);
 }
 .input-info-warning {
     color: var(--warning-darker);
+    font-weight: 500;
 }
```
Comment (manojsarna) [inline components/input/input.css:24]: looks good!!

File: components/list/list.css (hunk 1 above)
Comment (nisargpawade) [reply components/list/list.css:29]: less than 10 px

Comment (nisargpawade) [reply components/list/list.css:29]: less than 10 px

File: components/list/list.css [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.list-stacked-container li {
+    padding: 1rem;
```
Comment (manojsarna) [inline components/list/list.css:33]: looks good@!

File: components/text-utilities/text-utilities.css (hunk 2 above)
Comment (nisargpawade) [reply components/text-utilities/text-utilities.css:3]: less than 10 px

File: components/text-utilities/text-utilities.css [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+  }
+  
+  h3 {
+    font-size: 2.15rem;
+    margin: 4px 0;
+  }
+  
+  h4 {
+    font-size: 1.85rem;
+    margin: 4px 0;
```
Comment (manojsarna) [inline components/text-utilities/text-utilities.css:18]: do check calculations

File: components/text-utilities/text-utilities.html [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+        </div>
+        
+        <pre><code class="display-code language-html">
+            &lt;p class="p-lg"&gt;This is a Text&lt;/p&gt;
+            &lt;p class="p-md"&gt;This is a Text&lt;/p&gt;
+            &lt;p&gt;This is a Text&lt;/p&gt;
+            &lt;p class="p-sm"&gt;This is a Text&lt;/p&gt;
+            &lt;small&gt;This is small&lt;/small&gt;
+        </code></pre>
+
```
Comment (manojsarna) [inline components/text-utilities/text-utilities.html:117]: do use comments to elaoborate your code to the end developer reading you code!!

**Review by manojsarna — APPROVED** [review]
Overall code looks good!!!

you can merge!!

**Review by theVivek-kumar — APPROVED** [review]

File: components/list/list.css (hunk 1 above)
Comment (theVivek-kumar) [reply components/list/list.css:29]: looks good to me can be merged

File: components/alert/alert.html (hunk 4 above)
Comment (nisargpawade) [reply components/alert/alert.html:107]: `&lt;` - <
`&gt;` - >

File: components/text-utilities/text-utilities.html (hunk 9 above)
Comment (nisargpawade) [reply components/text-utilities/text-utilities.html:117]: i tried using class names which will give the idea

**Pull request merged**